README.md

```diff
@@ -1 +1,201 @@
-# Capstone-project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>CAPSTONE project site</b></h3> 
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Microverse Portfolio <a name="about-project"></a>
+
+> This a Microverse Project, for a fully responsive capstone project site.
+
+**Microverse Portfolio** is a fully responsive web application.
```

Comment:
- [ ] [OPTIONAL] I would suggest editing your project title and description to reflect this project as this will make your README relevant and more professional